components/builder-originsrv/src/server/handlers.rs

```diff
@@ -497,6 +497,70 @@ pub fn project_update(
     Ok(())
 }
 
+pub fn project_integration_create(
+    req: &mut Message,
+    conn: &mut RouteConn,
+    state: &mut ServerState,
+) -> SrvResult<()> {
+    let msg = req.parse::<proto::OriginProjectIntegrationCreate>()?;
+    match state.datastore.create_project_integration(&msg) {
+        Ok(()) => conn.route_reply(req, &NetOk::new())?,
+        Err(SrvError::OriginProjectIntegrationCreate(PostgresError::Db(ref db)))
+            if db.code == UniqueViolation => {
+            let err = NetError::new(ErrCode::ENTITY_CONFLICT, "vt:project-integration-create:1");
```

Comment:
indentation?

Reply:
It's weird but I think that's the way it's supposed to be since the if is part of the match branch